packages/angular-generator/src/expressions/jsx/attribute.ts

```diff
@@ -82,7 +80,7 @@ export class JsxAttribute extends BaseJsxAttribute {
         return 'ngStyle';
       }
 
-      if (ATTR_BINDING_ATTRIBUTES.indexOf(name) > -1) {
+      if (name.startsWith('aria-')) {
```

Comment:
I suggest using a `isAriaAttribute` function for a better understranding of the code.